src/ahp/verifier.rs

```diff
@@ -199,7 +202,8 @@ impl<F: PrimeField> AHPForR1CS<F> {
         // // This LC is the only one that is evaluated:
         // let inner_sumcheck = a_poly_lc - (b_lc * (gamma * &g_2_at_gamma + &(t_at_beta / &k_size))) - h_lc
```

Comment:
This comment should be changed as well (does  refer to Marlins classical sumcheck)